apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -165,8 +161,24 @@ const Courses = React.createClass({
           </div>
         )}
 
-        {!(isEnglish && isTeacher) && (
-          <div>
+        {/* Signed-in student in English */}
+        {(!isSignedOut && !isTeacher && isEnglish) && (
+          <ContentContainer
+            heading={i18n.teacherCourseHoc()}
+            description={i18n.teacherCourseHocDescription()}
+            isRtl={isRtl}
+            linkText={i18n.teacherCourseHocLinkText()}
+            link={`${codeOrgUrlPrefix}/learn`}
+            showLink={true}
+          >
+            <ProtectedStatefulDiv ref="hocCoursesFirstRow"/>
+          </ContentContainer>
+        )}
+
+        {/* Anything but a teacher or student in English.
+            That is: signed-out, or a student or teacher in non-English. */}
+        {(isSignedOut || !isEnglish) && (
+          <div id="all-courses">
```

Comment:
Oh, I just added this to make my own visual inspection of the DOM a bit easier.  I can take it back out again if not a good practice.